cmd/monaco/main.go

```diff
@@ -312,20 +312,20 @@ func getDownloadCommand(fs afero.Fs) cli.Command {
 			},
 			&cli.PathFlag{
 				Name:      "environments",
-				Usage:     "Yaml file containing environment to deploy to",
+				Usage:     "Yaml file containing environment to download from",
 				Aliases:   []string{"e"},
 				Required:  true,
 				TakesFile: true,
 			},
 			&cli.StringFlag{
 				Name:    "specific-environment",
-				Usage:   "Specific environment (from list) to deploy to",
+				Usage:   "Specific environment (from list) to download",
 				Aliases: []string{"s"},
 			},
 			&cli.StringFlag{
-				Name:    "downloadSpecificAPI",
+				Name:    "specific-api",
 				Usage:   "Comma separated list of API's to download ",
-				Aliases: []string{"p"},
+				Aliases: []string{"a"},
```

Comment:
sorry only just thought of this, please include `"downloadSpecificAPI"` and `"p"` as an alias for backward compatibility. 
We should remove them with v2 as this a big breaking change anyway (#679 679)